lava-v2-jobs-from-api.py

```diff
@@ -96,10 +124,9 @@ def main(args):
                         exit(1)
             if build['kernel_image']:
                 # handle devices without a DTB, hacky :/
+                build['dtb_dir_data'].append('qemu')
```

Comment:
Sorry about the defconfig bit, that's because I was testing on a subset of the whole KernelCI coverage (which is too big to keep testing in practice).  I'll drop this patch from this pull request and submit a cleaner one later on.

PS: Although it's not used in production yet, I am using only v2 boots with my dev system :)